extension/js/common/composer.ts

```diff
@@ -625,6 +624,9 @@ export class Composer {
     if (String(this.S.cached('input_to').val()).length) { // evaluate any recipient errors earlier treated as gentle
       await this.parseRenderRecipients('harshRecipientErrs');
     }
+    if (this.S.cached('icon_show_prev_msg').hasClass('progress')) {
```

Comment:
Good. I tend to save state in the DOM sometimes. I know the React community and others consider it a bad practice, but it has worked for me. Thanks for following the style of surrounding code!